src/material/form-field/_form-field-theme.scss

```diff
@@ -100,6 +100,7 @@
     // shows `body1` for text fields though, so we manually override the typography.
     // Note: Form controls inherit the typography from the parent form field.
     .mat-mdc-form-field,
+    .mdc-text-field__input,
```

Comment:
Should this be in the input theme?

Reply:
Also, I don't fully recall- but IIRC the input is supposed to just inherit typography from the wrapping form field structure. Same for custom controls

Reply:
Yeah you're right. I noticed a difference between the select and the input - but I think the correct fix is to inherit the `letter-spacing` and other properties not covered by `font`, updated the PR